docs/spl-docs/aggregate-functions/agg-functions.md

```diff
@@ -0,0 +1,186 @@
+
+	```spl
+	... | stats min(eval(latitude < 0)) AS min BY weekday
+	```
+
+- The following example finds the minimum `latency` value:
+
+	```spl
+	... | stats min(latency)
+	```
+
+- The following example displays a timechart of the minimum latency over time for each city:
+
+	```spl
+	... | timechart min(latency) BY city
+	```
+
+## **max(&lt;value&gt;)**
+This function returns the maximum value in a field.
+
+### Usage
+
+This function processes field values as numbers if possible, otherwise processes field values as strings. You can use this function with the `stats` and `timechart` commands. This command cannot be used on non-numeric fields.
+
+### Example
+
+- The following example returns the maximum value of the `latency` field:
+
+	```spl
+	... | stats max(latency)
+	```
+
+- The following example calculates the maximum value of the `latitude` field for locations in the northern hemisphere (latitude > 0) and groups the results by `weekday`:
+
+	```spl
+	... | stats max(eval(latitude > 0)) AS max BY weekday
+	```
+
+- The following example displays a timechart of the maximum latency over time for each city:
+
+	```spl
+	... | timechart max(latency) BY city
+	```
+
+## **range(&lt;value&gt;)**
+Returns the difference between the maximum and minimum values in a field.
+
+### Usage
+
+The values in the field must be numeric. You can use this function with the `stats` and `timechart` commands.
+
+### Example
+
+- The following command calculates the range of latency for each city, providing a simple measure of variability:
+
+```spl
+... | stats range(latency) AS LatencyRange BY city
+```
+
+### Detailed Example
+
+- For a more in-depth analysis, suppose you want to analyze network latency across different cities, focusing on peak hours (9 AM to 5 PM) during weekdays. You aim to identify cities with high latency variability and peak latency times:
+
+```spl
+... | where date_wday!="saturday" AND date_wday!="sunday"
+| where date_hour>=9 AND date_hour<=17
+| eval CityLatency=if(latency>200, "High", "Normal")
+| stats min(latency) AS MinLatency, max(latency) AS MaxLatency, range(latency) AS LatencyRange BY city
+| sort - LatencyRange
+```
+
+- The following example displays a timechart of the range of latency over time for each city:
+
+```spl
+... | timechart span=1h range(latency) BY city
+```
+
+## **sum(&lt;value&gt;)**
+This function returns the sum of the values in a field.
+
+### Usage
+
+You can use this function with the `stats` and `timechart` commands.
+
+### Example
+
+- To calculate the total latency for all events:
+
+	```spl
+	... | stats sum(latency)
+	```
+
+- To organize the results by a specific field, such as `city`:
+
+	```spl
+	... | stats sum(latency) AS "total latency" BY city
+	```
+
+- The following example displays a timechart of the sum of latency over time:
+
+	```spl
+	... | timechart sum(latency)
+	```
```

Comment:
nit: summing latency is probably not useful; the average latency would probably be more useful for making decisions.

Maybe summing something else would be more useful. For example, summing revenue from subscriptions per month per city